cmake/Runner.cmake

```diff
@@ -0,0 +1,38 @@
+# This source file is part of the Swift.org open source project
+#
+# Copyright (c) 2014 - 2023 Apple Inc. and the Swift project authors
+# Licensed under Apache License v2.0 with Runtime Library Exception
+#
+# See http://swift.org/LICENSE.txt for license information
+# See http://swift.org/CONTRIBUTORS.txt for Swift project authors
+
+function(SwiftTesting_GenerateDerivedSources)
+  message(STATUS "SwiftTesting: generating derived sources...")
```

Comment:
Nitpick: the package name is "swift-testing", not "SwiftTesting". Unless CamelCase is syntactically required, please use the former. :)

Reply:
Oh, I'm happy to change (or just completely remove) the `message()`.  The function itself is meant to follow CMake conventions like `FetchContent_MakeAvailable`, etc.

Reply:
The symbol names matching CMake style is fine—I'm just nitpicking the console message here.